authors/apps/notifications/models.py

```diff
@@ -0,0 +1,57 @@
+from authors.apps.profiles.models import Profile
+from django.db import models
+
+
+class NotificationManager(models.Manager):
+    
+    def create_notification(self, sender, receiver, message, action_link=None):
+        """Create and return a  notification"""
+        if sender not in Profile.objects.all():
+            raise TypeError('Notification must have a valid sender.')
+
+        if receiver not in Profile.objects.all():
+            raise TypeError('Notification must have a valid receiver')
```

Comment:
@malfahad the message is similar to that on line `9`. this can be extracted and carefully using `format` to fill in the different ending words.